### PR TITLE
Modern theme for colorpicker component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Design spec:

image

### CODE REVIEW

File: src/app/public/modules/colorpicker/colorpicker.component.spec.ts [hunk 1]
```diff
@@ -239,7 +273,7 @@ describe('Colorpicker Component', () => {
       fixture.detectChanges();
       tick();
       expect(component.colorpickerComponent.initialColor).toBe('#2889e5');
-      expect(component.colorpickerComponent.lastAppliedColor).toEqual({
+      expect(component.colorpickerComponent.lastAppliedColor as any).toEqual({
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/colorpicker/colorpicker.component.spec.ts:276]: ☝️ Not part of this PR, but it was causing the build to fail.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme)

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
I see this on the build result link:
image

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme)

**Comment by Blackbaud-AlexKingman** [comment]
> I see this on the build result link:
> image

Strange... Not sure why this change brought up that issue. I've added a null check to avoid the issue. This should be fixed now.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-modern-theme)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by Blackbaud-ToddRoberts — APPROVED** [review]
